cli/server.go

```diff
@@ -189,25 +200,25 @@ func server() *cobra.Command {
 			}
 
 			var (
-				ctxTunnel, closeTunnel = context.WithCancel(cmd.Context())
-				devTunnel              = (*devtunnel.Tunnel)(nil)
-				devTunnelErrChan       = make(<-chan error, 1)
+				ctxTunnel, closeTunnel = context.WithCancel(ctx)
+				devTunnel              *devtunnel.Tunnel
+				devTunnelErr           <-chan error
```

Comment:
Why are we moving to an un-buffered channel here?

Reply:
This is just so that the channel is accessible outside the `if`, the actual channel is returned by `devtunnel.New()`.

I thought it was weird that we're allocating a channel here since the `nil` channel blocks forever which is ideal, and informs the reader of our intent.